_component_design/cards.md

```diff
@@ -164,112 +151,84 @@ Cards are applied to the helix-grid and have a variable width and height based o
- ![]({{site.url}}/assets/images/components/content-areas/cards/card-use.png){:width="611px"}
-{% endfigure %}
-{% endcolumn %}
 </div>
 
 </section>
 
-## Variations
-
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Segemented card
-
-Cards the use all three of the underlying elements are referred to as a segmented card.
-
-- **The header:** may contain icons for card properties,card selection, duplication, deletion etc.
-
-- **The body:** contains the meat of the card content. It contains some combination of text, images, icons, graphics or visualizations that detail the underlying object it represents.
+### Loading
 
-- **The footer:** houses calls for a user to act on a card through the use of buttons or links.
+If an individual card or set of cards is loading, use the [Spinner Loading Indicator](http://helix.rax.io/components/loading-indicators.html) centered vertically and horizontally within the card body element.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [class:"image bg-light border"] %}
- ![]({{site.url}}/assets/images/components/content-areas/cards/card-segmented.png){:width="611px"}
+ ![]({{site.url}}/assets/images/components/content-areas/cards/card-loading.png){:width="469px"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
 </section>
 
-
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Open card
-
-Cards that opt to remove the header and footer elements and rely solely on the body element are referred to as open cards.
+### Empty card
 
-Open cards have less containing shapes than segmented cards and are therefore better for representing more focused or less complex objects.
+The empty card provides a space for a user to add a new card.
 
+The empty card button is based on the [chip](http://helix.rax.io/components/chips.html) component and is created by removing the fill of a chip, adding a dotted stroke to the containing shape, and overlaying a Helix plus icon (+).
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
 {% figure [class:"image bg-light border"] %}
- ![]({{site.url}}/assets/images/components/content-areas/cards/card-open.png){:width="469px"}
+ ![]({{site.url}}/assets/images/components/content-areas/cards/card-add.png){:width="469px"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-## States
+</section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Loading
-
-If an individual card or set of cards is loading, use the Spinner Loading Indicator centered vertically and horizontally within the card body element.
-
-
-{% endcolumn %}
+## Sample
 
```

Comment:
Where did the image, name and website come from for the Sirius Cole chacotaco.com sample image? Is there anything to be concerned about those? Just wanting to mitigate embarrassment/risk for HELIX and Rackspace when we take this site public in case something is not totally A-OK about it.